src/server/index.spec.tsx

```diff
@@ -88,6 +89,79 @@ describe(__filename, () => {
         expect(response.text).toContain('It Works.');
       });
 
+      it('sets production CSP and security headers', async () => {
+        const server = request(
+          createServer({
+            env: {
+              ...prodEnv,
+              REACT_APP_API_HOST: 'https://code-manager.addons.cdn.mozilla.net',
+            } as ServerEnvVars,
+            rootPath,
+          }),
+        );
+
+        const response = await server.get('/');
+
+        expect(response.status).toEqual(200);
+        expect(response.header).toHaveProperty('content-security-policy');
+
+        const policy = cspParser(response.header['content-security-policy']);
+        expect(policy['default-src']).toEqual(["'none'"]);
+        expect(policy['connect-src']).toEqual([
+          'https://code-manager.addons.cdn.mozilla.net',
```

Comment:
It should probably be `env.REACT_APP_API_HOST` here (`env` needs to be extracted out).